LocalSettings.php

```diff
@@ -4786,6 +4786,27 @@
 	'wmgFormatterUrlProperty' => [
 		'default' => false,
 	],
+	'wmgWikibaseClientNamespacesWithRepoAccess' => [
+		// default [] means all namespaces are included
+		'default' => [],
+		'gratisdatawiki' => [
+			NS_CATEGORY,
+			NS_PROJECT,
+			NS_TEMPLATE,
+			NS_HELP,
+			// NS_MODULE
+			828
+		],
+	],
+	'wmgWikibaseClientItemAndPropertySourceName' => [
```

Comment:
If we aren't adding to ManageWiki, these should stay as wiki overrides in Wikibase.php

Reply:
> If we aren't adding to ManageWiki, these should stay as wiki overrides in Wikibase.php

I better add them to ManageWiki, I was thinking of chatting with you regarding that, though.

Reply:
I added wmgWikibaseClientItemAndPropertySourceName and wmgWikibaseRepoLocalEntitySourceName

Reply:
Defaults need removed.

Reply:
> Defaults need removed.

Done